template/addons/trpc/protected-question-router.ts

```diff
@@ -0,0 +1,13 @@
+import { createProtectedRouter } from "./protected-router";
+
+export const protectedQuestionRouter = createProtectedRouter()
```

Comment:
nit: `Question` might confuse people as to "what this does", maybe `protectedExampleRouter`?